### ISSUE TITLE
PT1000 (Type -5) x 3 on octopus pro board with one being redundant

### ISSUE THREAD

**Issue by aju3169** [MarlinFirmware/Marlin#23669]
### Is your feature request related to a problem? Please describe.

I am using a BTT octopus pro which is working fine on my highly modified Ender 5 Pro. It is configured as I would like except that I would like to use 3 PT1000 temperature sensors (type -5).  My board is configured with X, Y, Z, Z2 , EO, E1 with TMC 2209 drivers.  The other two available drive slots (normally E2 & E3), I would like to use with the plugin BTT MX61865 for two more PT1000 sensors.  While another PT1000 is already available on the board itself.  Two of these sensors are for E0 and E1 while the last I want to be a redundant sensor for E0 (not E1).  Why?  Well I have a multiple extruders setup with the Wham Bam Mutant 2 system.  One of these is a dual extruder hence E0 and E1 while others are single extruders one of which has a redundant sensor.   For the other single extruders I can fake the redundant sensor by simply connecting the T0 sensor to the T0 and T2 ports on the board.  This allows me to have consistent wiring for all my extruder combinations with temperature sensors that can be used with accurate temperature measurement from 0 to 500 C.  So in summary I would like a PT1000 (type -5) redundant sensor  as E2 (T2), Pt1000 (type -5) on  E0 (T0), and PT1000 (type -5) on E1 (T1).  The marlin code currently doesn't let me use a PT1000 for a redundant sensor and I cannot have more than 2 MX61865 either.  The hardware is available but unfortunately the current marlin software doesn't allow it.   

I believe this would be very useful addition to the marlin firmware.  Can anyone help?  

### Are you looking for hardware support?

Octopus Pro with BTT TMC2209 on X, Y, Z, Z2, E0, E1 all in UART mode and BTT 61685 on remaining slots (normally E2 and E3) in SPI mode

Printer was originally an Ender 5 pro frame but now fitted with X and Y linear rails, dual Z axis, upgraded power supply 600W 24V, keenova hot bed (350W, 24V), BTT TFT70 screen, BTT TFT70 PI octoprint screen, Wham Bam Mutant 2 manual quick change tooling (for 7 different extruders optimised for different plastics - flexible, PLA, ABS, dual extrusion with soluble support, abrasive, etc)

### Describe the feature you want

In summary I would like a PT1000 (type -5) redundant sensor  (E2), Pt1000 (type -5) on E0, and PT1000 (type -5) on E1.   E2 being redundant for either E0 or E1 (for me I would only use E0).  Only two extruders defined in the configuration file - E0 and E1 so that should make E2 available as redundant sensor 

### Additional context

That's it - BTW the octopus pro pin file would need to be updated to allow all the PIN for T1, T2, and even maybe T3 to be replaced to match the PIN for the BTT 61865 in the event that type -5 sensor is selected for T1 and Tx (to match the E
